src/app.js

```diff
@@ -1,16 +1,40 @@
+'use strict';
+
+require('dotenv').config();
+const bodyParser = require('body-parser');
 const express = require('express');
+const mailer = require('./mailer');
 const path = require('path');
+
 const app = express();
-const bodyParser = require('body-parser');
-const PORT = process.env.PORT || 3000;
 
-app.post('/api/forms/contact', bodyParser.urlencoded({extended: true}), function(req, res) {
-  console.log(req.body);
-  res.sendStatus(204);
+const port = process.env.PORT;
+
+app.post('/api/forms/contact', bodyParser.urlencoded({ extended: true }), function(req, res) {
+  try {
+    const to = mailer.mailgun_admin_email;
```

Comment:
Only use snake_case for constants that are in all caps (i.e. ADMIN_EMAIL). Generally, the convention is for all constants to be in SNAKE_UPPERCASE, but generally in JavaScript variables appear in camelCase, so nobody would call you on using that.

There are a lot of different naming conventions. I like Google's style guide: https://google.github.io/styleguide/jsguide.html#naming-rules-common-to-all-identifiers